### ISSUE TITLE
Missing linux commands

### ISSUE THREAD

**Issue by p6002** [JanDeDobbeleer/oh-my-posh#3195]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Missing linux commands:
https://ohmyposh.dev/docs/installation/fonts

In general the whole site definitely needs to be rewritten, nothing can be found, nothing works. 
I tried on windows and linux. 

$ oh-my-posh font install
bash: /usr/local/bin/oh-my-posh: cannot execute binary file: Exec format error\

$ eval "$(oh-my-posh init bash)"
bash: /usr/local/bin/oh-my-posh: cannot execute binary file: Exec format error

### Theme

a

### Log output

```Shell
$ oh-my-posh debug
bash: /usr/local/bin/oh-my-posh: cannot execute binary file: Exec format error
```

**Comment by JanDeDobbeleer** [comment]
@p6002 we accept pull requests. And thanks for being so appreciative of all the work that has been put into all of this.

On the topic, the linux manual installation guide assumes `amd64`, whereas you're probably using an `arm64` device. I'll add that context so people are aware of that.

If so, download the correct binary. We have arm, arm64 and amd64. Assuming arm64:

```
sudo wget https://github.com/JanDeDobbeleer/oh-my-posh/releases/latest/download/posh-linux-arm64 -O /usr/local/bin/oh-my-posh
sudo chmod +x /usr/local/bin/oh-my-posh
```

**Comment by p6002** [comment]
Your command works, but the appearance of the terminal has not changed. 
Okay I'm already lost, I installed some 10 files from your site and now I don't even know how to remove it. 
If the installation procedure looks like: app, font, theme, then make the page point 1 app, add download links, 2 font, give download commands, etc. 
Meanwhile, the instructions on the page break off in several places, not allowing the installation to complete. The project doesn't even have a subreddit for any help. I'm not a Linux programmer to figure out the rest of the tutorial. But with a good manual, you can do anything, as I recently did grafan+cadvisor+node-exporter on synology and everything works. 
Fix the site, for you it's probably all ok,but you didn't make the site for yourself (I guess) :)

**Comment by JanDeDobbeleer** [comment]
@p6002 once installed, you only need to really read the steps [here](https://ohmyposh.dev/docs/installation/prompt). What you wrote is correct. These pages have been adjusted over the course of years based on user input, and it's been a while since we got feedback that things weren't clear. It's already structured for download, font and configure which are the logical steps to take.

**Comment by p6002** [comment]
I found some video on yt and managed to get it partially working on win11, but the letters have errors.
https://i.ibb.co/2jQhg31/s20221213-183457.png

On your site the installation point is ok, but later in the font section there are no instructions for Linux, 
in the next Prompt there are "too many" options to choose from and none works for my rpi and synology. 
After all it's just a theme, I should type the command and it it should work by itself :)

**Comment by JanDeDobbeleer** [comment]
@p6002 define "too many options"

The font issue is either linked to the font, or terminal font settings (spacing/width). Oh-my-posh can't influence the actual rendering.

**Comment by p6002** [comment]
Okay my powershell is no longer working :)

I have these options in mind:

powershell
cmd
zsh
bash
fish
nu

**Comment by JanDeDobbeleer** [comment]
@p6002 did you see the big green block above those options?

![image](https://user-images.githubusercontent.com/2492783/207423136-b76737ad-a116-4155-b0d1-bc0dc2b6a7b9.jpeg)

If you have no idea what a shell is, I suggest you look into that as that goes beyond the responsibility of oh-my-posh.

**Comment by p6002** [comment]
Yes it is not for me, problems with posh, with fonts, errors in powershel, etc. 
Besides, rpi architecture is arm7l and synology server is x86_64. 
Anyway I would like to remove it from windows and the Uninstall-Module oh-my-posh command works, but does not remove it. How to remove it?

It looks like that:
https://i.ibb.co/GQ8XYNZ/u-20221213-205906.png

**Comment by JanDeDobbeleer** [comment]
@p6002 why did you install the module? That one's no longer supported... Depending on how you installed, use the same method to uninstall.

BTW, that prompt looks perfect other than not having configured the font in your terminal. Which is also listed in next steps.

![image](https://user-images.githubusercontent.com/2492783/207433454-b6ece5ec-c943-4b61-a5a5-83bb3e944663.jpeg)

**Comment by p6002** [comment]
I installed the font, but the font is too tight and the letter m overlaps. 
https://i.ibb.co/HTJGQTs/s-20221213-203742.png

But I don't need it for windows, more ubuntu, synology, rpi. 
I just thought it works like a theme, not that I have to install different version everywhere, some different architectures, fonts, thems, json mods etc.
I lost an entire evening to this after work XD
Fortunately I modified powershell 7 from windows store, and the built-in one works normally. 
(however, after reinstalling powershell 7, the skin is still visible)
